doc/changelog.md

```diff
@@ -1,3 +1,9 @@
+# v0.13.0 -> v0.x.x
+
+## Bugfixes
+
+- When a preconfigured task is removed from a supertask and its originating from an import, it should be deleted (issue #865).
```

Comment:
The description is not clear enough.

Suggestion:

`When deleting a supertask that was created from an import, pretasks that were removed from this supertask should also be deleted.`

Reply:
The text is now changed to your suggestion.